javatools_unicode/src/main/java/org/xvm/tool/BuildUnicodeTables.java

```diff
@@ -212,10 +212,16 @@ void writeResult(final String name, final String[] array) throws IOException {
         }
 
         final var sb = new StringBuilder();
-        sb.append(name).append(": [index] \"str\" (freq) \n--------------------");
+        sb.append(name)
+            .append(": [index] \"str\" (freq) \n--------------------");
```

Comment:
This line seems to be misaligned. Visually, it looks much neater if all the ".append" sit right under each other

Reply:
It's not misaligned. The indentation and the continuation indentation are four (4) spaces. All code standards I have gone through, including the language and platform agnostic ".editorconfig" global flags for all languages maintain the concept of indentation with the special case continuation indentation. While it can be another indentation level (e.g. continuations are indented 2 indent steps == 8, for an indentation of 4, or one indent step == 4). We can never have a continuation indentation that is not a multiple of the indent depth for any line. I mostly see (indent space) + (indent space), or (indent space) + 2 * (indent space) (e.g Google/Palantir) for continuation indentations like this.

Maintaining a context sensitive indenter that would check the surrounding code and do "appropriate" alignment based on lex tokens and/or control flow "in the vicinity" is very likely a path to madness, and I'm not surprised that no one seems to have done that, as that is what it takes. 

Preserving while space between variable, equals sign and value in a sequence of declarations is fine, and another thing altogether. Checkstyle makes no demands for that (and I'm sure we can figure out how to get IntelliJ to just do what our checkstyle says, or at least find the button that doesn't gobble whitespace, but we DO pass the code style check no matter if we have that declaration list variable alignment whitespace or not).

Here, though, we you are talking about changing indentation continuation depth, which is not something we want to do. In Python, it will even be a syntax error. Even though we are not Python, the indentation levels are *never* different and *never* depend on context. It is considered first priority that control flow is indented correctly and associated with its nesting level, no matter where it is. Aligning the dot back would suddenly cause a two space indentation in a world that uses indentation steps of 4.

I am probably not alone in thinking it's harder to read, especially when I look at more lines than above at the same time. The picture my brain uses the indents to create, basically a control flow map, immediately red-flags that continuation indent for me and tells me "that one doesn't look quite right, does it?" if it's not placed at a multiple of the global indentation space (n * 4).

Reply:
As you can see, we already disagree on the desired look and what "looks right" for just one place in one file. Imagine what happens when we try to apply this to 300,000 lines of code that Cam and I wrote earlier.

I believe that things like that should be kept as an author of the source choose. If you look at the original submission by Cam, the code looked like this:

            sb.append("\n[")
              .append(index)
              .append("] \"")
              .append(entry.getKey())
              .append("\" (")
              .append(entry.getValue())
              .append("x)");


Therefore, I strongly believe it should remain to look like that

Reply:
This type of thing doesn't occur _that_ often, but "sb" and "buf" (with the style of fluency seen above) are not uncommon. I do imagine that it's quite a complicated "rule" to explain (and/or codify). We generally use 8 space indent for trailing lines, so it would look like this:
```
    sb.append("\n[")
            .append(index)
            .append("] \"")
            .append(entry.getKey())
            .append("\" (")
            .append(entry.getValue())
            .append("x)");
```

The visual benefit of the former is fairly easy to defend, from my POV.

Reply:
random semi-unrelated.  I'm tons and tons of short complex printing, similar to above, into a StringBuilder.
I got one with very short print names specifically for this kind of thing, plus some formatting variants:
`sb.p("\n[").p(index).p("] \"").p(entry.getKey()).p("\" (").print(entry.getValue()).p(x)");`
OR 
`sb.fmt("\n[%0] \"%1\" (%2x)",index,entry.getKey(),entry.getValue());`